code/game/objects/items/robot/robot_items.dm

```diff
@@ -202,5 +206,6 @@
 			M.ear_deaf += 10
 			M.Knockdown(7)
 			M.Jitter(30)
+			add_logs(M, "alarmed by an emmaged \the [src.name]", admin = FALSE)
```

Comment:
1. Unlike the earlier ones, remove the `\the` here, but leave it as `[src.name]` (or just `[name]`)
2. >`emmaged`